playground/tailwind.config.ts

```diff
@@ -0,0 +1,16 @@
+import type { Config } from 'tailwindcss';
+import form from '@tailwindcss/forms';
+import plugin from 'tailwindcss/plugin';
+
+export default {
+	content: ['./app/**/*.{js,jsx,ts,tsx}'],
+	theme: {
+		extend: {},
+	},
+	plugins: [
+		form,
+		plugin(function ({ addVariant }) {
+			addVariant('touched', '&[data-conform-touched="true"]');
```

Comment:
@edmundhung is this an artifact from previous versions of conform? maybe we can remove?

Reply:
We can remove it. It's a legacy thing pre v1.